clang/lib/CIR/Lowering/ThroughMLIR/LowerCIRToMLIR.cpp

```diff
@@ -42,8 +46,14 @@
 #include "clang/CIR/Dialect/IR/CIRTypes.h"
 #include "clang/CIR/LowerToMLIR.h"
 #include "clang/CIR/Passes.h"
+#include "llvm/ADT/STLExtras.h"
 #include "llvm/ADT/Sequence.h"
+#include "llvm/ADT/SmallVector.h"
 #include "llvm/ADT/TypeSwitch.h"
+#include "llvm/Analysis/RegionInfo.h"
+#include "llvm/IR/Type.h"
+#include "llvm/IR/Value.h"
+#include "llvm/Support/Casting.h"
```

Comment:
nit: does the new adding include lines are required?

Reply:
These files are generated by clangd’s auto-completion, and I believe they are not necessary

Reply:
Hello, I have moved the CIRWhileOpLowering to LoweringSCFLoop.cpp , can you review it?

Reply:
Could we remove the un-required including lines?